app/services/katello/pulp3/repository/yum.rb

```diff
@@ -305,8 +309,15 @@ def copy_content_from_mapping(repo_id_map, options = {})
             modular_filters = ContentViewModuleStreamFilter.where(:id => dest_repo_map[:filter_ids])
             filters.flatten!.compact!
 
+            errata_filters = ContentViewErratumFilter.where(:id => dest_repo_map[:filter_ids])
+
+            # Only filter out RPMs & modules via an erratum filter if they're not included in other allowed errata.
+            # Begin this by calculating all the errata that should be copied.
+            # If there are exclude filters, send the errata to the relevant methods to ensure their contents aren't excluded.
+            errata_to_ensure_complete, all_excluded_errata = errata_exclusion_helpers(errata_filters, source_repo_ids)
```

Comment:
is 'errata_to_ensure_complete' really just a list of all errata to include?   Trying to think of a name that would be more clear